skycatalogs/objects/snana_object.py

```diff
@@ -1,4 +1,5 @@
 import bisect
+from math import isnan
```

Comment:
It probably doesn't matter, but since `numpy` is already being imported, I would just use `np.isnan` instead of `math.isnan`.